actionpack/test/controller/redirect_test.rb

```diff
@@ -604,9 +604,8 @@ def test_url_from_fallback
   def test_redirect_to_instrumentation
     payload = nil
 
-    subscriber = proc do |*args|
-      event = ActiveSupport::Notifications::Event.new(*args)
-      payload = event.payload
+    subscriber = proc do |_, _, _, _, event_payload|
+      payload = event_payload
     end
```

Comment:
What about:

```suggestion
    subscriber = proc do |event|
      payload = event.payload
    end
```

Or even:

```suggestion
    subscriber = proc { |event| payload = event.payload }
```

The four ignored block arguments are noisier visually than manually building the event, imo 🤷

Reply:
I agree and had that at first.
Then I saw similar test had this pattern instead.
I'll update those as well.

Reply:
Ah, sorry. I didn't notice the example of moving it to a single line, which is even better IMO.

Reply:
No worries, it's a very minor difference 🙂

Reply:
😄